src/libs/blueprint/conduit_blueprint_mesh_examples.cpp

```diff
@@ -3585,7 +3585,273 @@ polychain(const index_t length, // how long the chain ought to be
     }
 }
 
+//-----------------------------------------------------------------------------
+void polytess_3d(conduit::index_t nlevels, Node &res)
```

Comment:
The interface split between `polytess` and `polytess_3d` seems to go against the established integrated 2D/3D interface convention of the other `examples` functions (e.g. `basic`, `braid`, etc.). Is there any way that `polytess_3d` integrated into `polytess` rather than being a separate function? I could envision using the `nz` variable seen in the other `examples` functions in a similar way here, e.g.:

```
// nz == 0/1 -> 2D
// nz == 2+ -> 3D (variable number of planes instead of just z=0, 1)
void polytess(index_t nlevels, index_t nz, Node &res);
```

Reply:
ok, so this would allow for users to create not just a single "sheet" of polytess, but stack sheets arbitrarily high?

Reply:
Yes, that's correct. To be honest, this is more of an aesthetic consideration rather than a practical one, but I do think having a greater level of interface uniformity is a good thing to aim form.